src/vs/workbench/browser/parts/editor/multiEditorTabsControl.ts

```diff
@@ -1198,6 +1213,34 @@ export class MultiEditorTabsControl extends EditorTabsControl {
 		}
 	}
 
+	private isHeadOfTab(e: DragEvent, tab: HTMLElement): boolean {
```

Comment:
I find the method name a bit hard to understand, I think you want to indicate if the user drags over the left or right side of the tab? Is there maybe a better name for that? Maybe `getTabDragOverLocation(): left/right` or something similar.